README.md

```diff
@@ -1,2 +1,12 @@
-# NoteApp_Gavin-Emeka
-Week 2 Task - Beginners with Experience.
+# NoteApp_Gavin-Emeka
+### Week 2 Task - Beginners with Experience.
+
+## Of all the Ups and Downs that this task has taken me through, My greatest Joy is that I could  do something genuinely without using ChatGPT. I guess its a big one for me.
+
+## QST.1: I was able to create the Parent Class, NOTE, and used constructors to set up all the attributes that the class could have. I was also able to create differnt Object using the class.
+
+## QST.2: I was able to extend my base class, to create subclasses that would inherit its Properties.The TextNote and the ReminderNote.
+
+#### Couldn't attempt the other questions due to poor knowledge of it, and the time left, until submission.
+
+### I'm very much open to reviews and corrections, on the next steps better steps to take.

```

Comment:
Your readme is suppose to relay information about how your project works and how to use your project.